packages/@aws-cdk/aws-codebuild/lib/project.ts

```diff
@@ -654,6 +661,7 @@ export class Project extends ProjectBase {
   private readonly source: ISource;
   private readonly buildImage: IBuildImage;
   private readonly _secondarySources: CfnProject.SourceProperty[];
+  private readonly _secondarySourceVersions: CfnProject.ProjectSourceVersionProperty[];
```

Comment:
This should be called '_secondarySource**s**Versions', to be consistent with `_secondarySources`.

Reply:
The [CloudFormation property](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-codebuild-project.html#cfn-codebuild-project-secondarysourceversions) is called "SecondarySourceVersions." Wouldn't it be better to be consistent with the CF names?